common/buildcraft/lib/block/BlockBCBase_Neptune.java

```diff
@@ -42,22 +41,30 @@
     public static final IProperty<EnumFacing> PROP_FACING = BuildCraftProperties.BLOCK_FACING;
     public static final IProperty<EnumFacing> BLOCK_FACING_6 = BuildCraftProperties.BLOCK_FACING_6;
 
-    /** The tag used to identify this in the {@link TagManager} */
+    /** The tag used to identify this in the {@link TagManager}. Note that this may be empty if this block doesn't use
+     * the tag system. */
     public final String id;
 
+    /** @param id The ID that will be looked up in the {@link TagManager} when registering blocks. Pass null or the
+     *            empty string to bypass the {@link TagManager} entirely. */
```

Comment:
so for addon makers (like @faceofcat ) they are no longer required to also register things in the tagmanager?

Reply:
Yes.